Libraries/StyleSheet/NativeColorValueTypes.android.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @format
+ * @flow strict-local
+ */
+
+'use strict';
+
+import type {ColorValue, ProcessedColorValue} from './ColorValueTypes';
+
+export opaque type NativeColorValue = {
+  resource_paths?: Array<string>,
+};
+
+export const PlatformColor = (...names: Array<string>): ColorValue => {
+  return {resource_paths: names};
+};
+
+export const IOSDynamicColor = (
```

Comment:
It doesn't feel feasible to have the other platform type definitions in this one. Wouldn't that mean that every platform would have to have the interface for all other platforms in the ecosystem? If any of those platforms want to change the API it would it require changing all the other platforms?

This API is the part I'm most concerned about figuring out in this PR.

Reply:
Whoops.  This should not be here -- I removed it.  
It was a left over from when I was making some test code testing a hypothetical Android specific ColorValue creation method.

Reply:
I didn't mean for this to be here -- removed it.
It was a left over from an experiment I did to implement a hypothetical Android specific `ColorValue` method.